mev/mev/base_settings.py

```diff
@@ -534,3 +529,22 @@
 # END settings/imports for public data indexing
 ###############################################################################
 
+# Change the LOGLEVEL env variable if you want logging
+# different than INFO:
+LOGLEVEL = os.environ.get('LOGLEVEL', 'info').upper()
```

Comment:
Should the default value be `INFO`?

Reply:
Is this with respect to a) lower vs. uppercase or b) whether the default should be INFO, instead of WARNING, etc.?

Reply:
Regarding the case of the default value for `get()`: this makes it more explicit that uppercase values are expected.